### ISSUE TITLE
Incorrect user/password after database restore

### ISSUE THREAD

**Issue by xaro1** [kimai/kimai#581]
After fresh install + database restore, I'm unable to log in. ALL users are rejected (also admin)

<pre>Access denied!
Wrong Username and/or Password</pre>

And now... What ?!?

**Comment by xaro1** [comment]
Okay, I've get the default "changeme" password hash from fresh install and put in my restored database. I can now log as admin with "changeme" password and can now change users password from admin panel (after some code modifications see #582). I'm too old to play with that stupid game...

**Comment by kevinpapst** [comment]
So, what did you do exactly? Did you restore your original autoconf.php with the password salt?

**Comment by xaro1** [comment]
Old version: git master around 12/2015
New version: git master from today morning

I have copied autoconf.php from old version to new fresh install and restored database directly from an automated mysql dump (never used backup utility)

**Comment by kevinpapst** [comment]
Okay, thanks for the info. I will try that with a couple of my old installations. 
Did the upgrade process step in?

**Comment by xaro1** [comment]
Yes, called updater/updater.php after database restore but directly forwarded to login screen.

**Comment by kevinpapst** [comment]
I wasn't able to reproduce the problem on multiple installations. 
I assume you don't have the debug log from the update?

**Comment by kevinpapst** [comment]
No feedback. Closing until someone else confirms it

**Comment by Stadly** [comment]
I just experienced the same thing. I had version 0.9.2 and updated to 1.0.1. Afterwards nobody was able to log in, except from admin/changeme. I do have the logfile.txt entries from the update available, if they can be of any help.

**Comment by simonschaufi** [comment]
As Kevin said, the salt in the autoconf.php file has to be the same like before. Otherwise no login is possible. During the update process, no passwords are updated, so it has to work!

**Comment by Stadly** [comment]
The steps I did were:
1. Make sure that includes/autoconf.php did not exist in the new version
2. Copy the new version over the old version, replacing any already existing files
3. From the log file I see the following actions:
  1. -- begin update
  2. -- begin backup
  3. -- backup finished
  4. several "-- update to rxxxx"
  5. -- update finished
  6. -- begin install
  7. -- showing install error

I guess I should not have performed an installation. Probably the salt was replaced during the installation.

I found it a bit hard to follow the [update instructions](http://www.kimai.org/documentation/installation/updates.html), as it doesn't say which urls to visit. I remember correctly, nothing happened when I went to the Kimai root, so I visited the installer directory to see if that would do the trick.

**Comment by kevinpapst** [comment]
Oh, I see this is an old and closed thread and your problem is different. @Stadly please open a new issue if replacing autoconf.php doesn't help.
